src/tests/mediaPresenter.test.ts

```diff
@@ -0,0 +1,96 @@
+import { MediaPresenter } from "../MediaPresenter";
+describe("media presenter", () => {
+  let presenter: MediaPresenter;
+  const imageFile = new File(["(⌐□_□)"], "blank.png", { type: "image/png" });
+  const videoFile = new File(["(⌐□_□)"], "blank.mp4", { type: "video/mp4" });
+  const textFile = new File(["(⌐□_□)"], "blank.pdf", {
+    type: "application/pdf",
+  });
+  beforeEach(() => {
+    presenter = new MediaPresenter();
+  });
+  describe("addFile", () => {
+    it("should append a file element to the files list", () => {
+      expect(presenter.filesLength).toEqual(0);
+      presenter.addFile(imageFile);
+      expect(presenter.filesLength).toEqual(1);
+      expect(presenter.media[0].file).toEqual(imageFile);
+    });
+
+    it("should return false if file is not an image or a video", () => {
+      let returnVal = presenter.addFile(textFile);
+      expect(returnVal).toBe(false);
+    });
+
+    it("should not append a file element if file is not an image or a video", () => {
+      expect(presenter.filesLength).toEqual(0);
+      presenter.addFile(textFile);
+      expect(presenter.filesLength).toEqual(0);
+    });
+    it("should return true if a video file is added", () => {
+      let returnVal = presenter.addFile(videoFile);
+      expect(returnVal).toBe(true);
+    });
+    it("should return true if an image file is added", () => {
+      let returnVal = presenter.addFile(imageFile);
+      expect(returnVal).toBe(true);
+    });
+  });
+
+  describe("removeFile", () => {
+    it("should remove a file element from the files list", () => {
+      expect(presenter.filesLength).toEqual(0);
```

Comment:
Can omit this first `expect`, because that tests the add behaviour, which is not part of this test. 
Same in your other tests